### ISSUE TITLE
[receiver/splunkhec] support trace signal

### ISSUE THREAD

**Issue by antonjim-te** [open-telemetry/opentelemetry-collector-contrib#35953]
### Component(s)

receiver/splunkhec

### Is your feature request related to a problem? Please describe.

The https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter supports traces but not the receiver. And it is a requirement for us.

### Describe the solution you'd like

[receiver/splunkhec] support trace signal

**Comment by atoulme** [comment]
How would that look like? Please show an example of trace as a HEC event.

**Comment by pellared** [comment]
> And it is a requirement for us.

Why do you actually need it? Why cannot you simply send traces using OTLP or any other protocol?

**Comment by vutnguye-te** [comment]
> > And it is a requirement for us.
> 
> Why do you actually need it? Why cannot you simply send traces using OTLP or any other protocol?

@pellared Our otel-collector pipeline supports exporting traces to SplunkHec. For our integration test, we plan to use a SplunkHec trace receiver to verify that data is being exported accurately.

**Comment by atoulme** [comment]
I see, you want to validate trace exports via a splunkhecreceiver. We support traces in the splunkhecexporter by marshaling them to JSON and making the event field of the HEC event. You want to read that event field and unmarshal the trace from it, correct?

**Comment by vutnguye-te** [comment]
Yes, we need to validate trace exports via a splunkhecreceiver. I am not sure how json marshalling can help us. Can you please elaborate a bit more? Thanks.
